demo/src/profile.ts

```diff
@@ -19,7 +19,7 @@ async function main() {
     const runtimeVersion = api.runtimeVersion;
     const runtimeType = runtimeVersion.specName.toString();
 
-    const stashUri = process.env.STASH_URI || '//Alice'; // Default to Alice for dev chains
+    const stashUri = '0xa456bbffe088bfe39695d603ca8a85c70b866a4633619e70831f7bf16491d74b//1'; // Default to Alice for dev chains
```

Comment:
Can this be something else? like `//demo//1` And lets load the account of that secret.

Reply:
ack